clang/lib/Driver/ToolChains/MSP430.cpp

```diff
@@ -161,6 +161,7 @@ void MSP430ToolChain::addClangTargetOptions(const ArgList &DriverArgs,
                                             ArgStringList &CC1Args,
                                             Action::OffloadKind) const {
   CC1Args.push_back("-nostdsysteminc");
+  CC1Args.push_back("-D__ELF__");
```

Comment:
If this is necessary, then it should be properly presented as a builtin define.

Reply:
Fixed.